frontend/app-development/features/administration/components/MainContent.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Button, TextArea, TextField } from '@digdir/design-system-react';
+import { Button, TextArea, LegacyTextField as TextField } from '@digdir/design-system-react';
```

Comment:
I don't know if it's still relevant, but I remember that we already discussed using aliases for Legacy components and we decided to avoid it to make it clear that the code was using the Legacy version of the component:
https://github.com/Altinn/altinn-studio/pull/10840#discussion_r1299939395

Reply:
Thats true! I'll change it 😄